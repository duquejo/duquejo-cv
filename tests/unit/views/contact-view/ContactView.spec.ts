import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";

import ContactView from '../../../../src/views/ContactView.vue';

describe('+ ContactView tests', () => {

  it('Should match the snapshot', () => {
    const wrapper = mount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});