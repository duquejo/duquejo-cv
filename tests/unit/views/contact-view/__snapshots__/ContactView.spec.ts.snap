// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`+ ContactView tests > Should match the snapshot 1`] = `
"<article>
  <section>
    <h1 class="text-h1">Contact</h1>
  </section>
  <section>
    <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iste debitis, ut error unde dicta nobis assumenda alias, voluptas porro, quidem quae a ea sit repudiandae quod illum incidunt tempora. Incidunt rem dolor consequatur nihil porro numquam, corrupti laborum cum nam eum sed odit eos odio, necessitatibus quas officia quia pariatur? </p>
    <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora perferendis deserunt aut non esse modi vero voluptatum, architecto adipisci, ex ipsum cum expedita odio provident! Nemo, blanditiis corrupti iure ipsa quasi laboriosam, perspiciatis accusamus repellat repellendus tenetur debitis autem dicta, officiis voluptates. Impedit modi ab adipisci odit enim debitis deleniti perspiciatis harum tempore magni quidem ducimus, nisi unde asperiores, voluptatibus distinctio. Vel, tempore quis delectus ducimus iure aperiam optio veritatis quibusdam quas quaerat eos eum at impedit vitae recusandae modi animi. Quos, dolorum dicta vel eaque, sit ab aperiam, debitis temporibus architecto alias maiores voluptate nisi esse culpa aliquam enim. </p>
  </section>
</article>"
`;
